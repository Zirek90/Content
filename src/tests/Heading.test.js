import React from "react";
import renderer from "react-test-renderer";
import HeadingComponent from "../components/Heading";

describe("HeadingComponent", () => {
  it("renders correctly", () => {
    const props = "Top article title";
    const tree = renderer.create(<HeadingComponent heading={props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
