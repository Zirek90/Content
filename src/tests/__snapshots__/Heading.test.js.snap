// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadingComponent renders correctly 1`] = `
<div
  className="sc-gtssRu ezxSyx"
>
  <h1
    className="sc-iCoHVE dNHPew"
  >
    Top article title
  </h1>
</div>
`;
